src/Containers/MovieListPage/MovieListPage.jsx

```diff
@@ -1,77 +1,64 @@
-import { useState } from 'react';
 import { GenreSelect } from '../../Components/GenreSelect/GenreSelect';
-import { SearchForm } from '../../Components/SearchForm/SearchForm';
 import { genres, sortOptions } from '../../constants';
-import { MovieDetails } from '../../Components/MovieDetails/MovieDetails';
 import { MovieTile } from '../../Components/MovieTile/MovieTile';
 import { SortControl } from '../../Components/SortControl/SortControl';
-import { Dialog } from '../../Components/Dialog/Dialog';
-import { MovieForm } from '../../Components/MovieForm/MovieForm';
 import styles from './MovieListPage.module.css';
-import { useFetchMovies } from '../../customHooks/movies';
+import { useFetchMovies } from '../../customHooks/moviesHooks';
+import { Outlet, useSearchParams } from 'react-router-dom';
 
-export function MovieListPage() {
-  const [selectedGenre, setSelectedGenre] = useState(genres[0]);
-  const [searchQuery, setSearchQuery] = useState('');
-  const [selectedSortOption, setSelectedSortOption] = useState(sortOptions[0].value);
-  const [selectedMovie, setSelectedMovie] = useState();
+const defaultQuery = {
+  search: '',
+  genre: genres[0],
```

Comment:
For consistency, `genres` like `sortOptions` should probably also contain objects of `{value: string, label: string}` type.